revolv/payments/tests/models/test_payment_instrument_type.py

```diff
@@ -1,12 +1,16 @@
 from django.test import TestCase
-from revolv.payments.models import INSTRUMENT_PAYPAL, PaymentInstrumentType
+from revolv.payments.models import (INSTRUMENT_CHECK, INSTRUMENT_PAYPAL,
+                                    INSTRUMENT_REINVESTMENT,
+                                    INSTRUMENT_REPAYMENT,
+                                    PaymentInstrumentType)
 
 
 class PaymentInstrumentTypeTest(TestCase):
 
-    fixtures = ['supported_instruments.json']
-
     def test_get_payment_instrument_types(self):
 
-        self.assertEqual(1, len(PaymentInstrumentType.objects.all()))
+        self.assertEqual(4, len(PaymentInstrumentType.objects.all()))
         self.assertEqual(INSTRUMENT_PAYPAL, PaymentInstrumentType.objects.first().name)
+        self.assertEqual(INSTRUMENT_REINVESTMENT, PaymentInstrumentType.objects.get(id=2).name)
```

Comment:
@noahsark769 the IDs work mainly because those values are in there because of a data migration so we know what ID they should be. Nothing else should write to that table. If this test breaks than something must be wrong.